test/helpers.bash

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Root directory of integration tests.
+INTEGRATION_ROOT=$(dirname "$(readlink -f "$BASH_SOURCE")")
+
+# Test data path.
+TESTDATA="${INTEGRATION_ROOT}/../testdata"
+
+# Root directory of the repository.
+OCID_ROOT=${OCID_ROOT:-$(cd "$INTEGRATION_ROOT/../.."; pwd -P)}
+
+# Path of the ocid binary.
+OCID_BINARY=${OCID_BINARY:-${OCID_ROOT}/ocid}
+# Path of the ocic binary.
+OCIC_BINARY=${OCIC_BINARY:-${OCID_ROOT}/ocid}
+# Path of the runc binary.
+RUNC_BINARY=${RUNC_BINARY:-${OCID_ROOT}/runc}
+
+PATH=$PATH:$BATS_TMPDIR
+
+# Run ocid using the binary specified by $OCID_BINARY.
+# This must ONLY be run on engines created with `start_ocid`.
+function ocid() {
+	sudo "$OCID_BINARY" "$@"
```

Comment:
Is it a good idea to use sudo within bats tests? Shouldn't we require that the tests be run as root (as with the runC tests)? Or do you want parts of ocid to work without privileges when testing?